CONTRIBUTING.md

```diff
@@ -0,0 +1,139 @@
+
+In addition to the requirements described in the upstream contributing guide, there are additional prerequesites you must have when contributing to this repository:
+
+- **Moderate knowledge of the English language:** You must have a good enough understanding of the English language to be able to decipher the meaning of a page while translating. (For simple typo fixes, this is not required.)
+- **Fluency in the downstream locale:** You must be able to fluently speak the language you are contributing to.
+
+### Simple changes
+
+If you want to make a small change like fixing a typo, the GitHub UI is the easiest way to get started.
+If you've found a typo on the Simplified Chinese [JavaScript landing page](/zh-CN/docs/Web/JavaScript), for example, you can propose a fix as follows:
+
+1. Sign in to [GitHub](https://github.com/)
+2. Navigate to [https://github.com/mdn/translated-content](https://github.com/mdn/translated-content)
+3. Find the source file `files/zh-cn/web/javascript/index.md`
+4. Click the edit (pencil) button
+
+From there, the GitHub UI will walk you through the rest by creating a [fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo) and a branch to commit your changes to.
+After you have made changes to your branch, the goal is to open a [pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests) for your changes to be incorporated.
+
+A pull request represents the work you want to be reviewed, approved, and merged into the `main` branch of the MDN repository.
+See the [Creating a pull request](#creating-a-pull-request) for more details on creating and handling pull requests successfully.
+
+If you're not certain of the changes that you want to make, [get in touch with us](/en-US/docs/MDN/Community/Communication_channels)!
+
+> **Note:** You can click the **View the source on GitHub** or **Edit the page on GitHub** link at the bottom of an MDN page to jump directly to the page source on GitHub.
+
+### Forking and cloning the repository
+
+If you want to make changes to more than one file, the GitHub UI is not very efficient because you have to make separate pull requests for each file you want to change.
+Instead of using the GitHub UI, you need to use `git` or a client like [GitHub Desktop](https://docs.github.com/en/get-started/using-github/github-desktop) or [GitHub CLI](https://docs.github.com/en/github-cli/github-cli/about-github-cli).
+
+#### Set up `mdn/content` repo
+
+First, you will need to set up the `mdn/content` repo locally. If you do not plan to contribute to upstream content, you will not need to fork it as described in its [Contributing guide](https://github.com/mdn/content/blob/main/CONTRIBUTING.md#forking-and-cloning-the-repository); all you need to do is clone it by running the following command:
+
+```bash
+git clone git@github.com:mdn/content.git
```

Comment:
Should we use `https` protocol? as we have to setup SSH key for using SSH protocol. If someone prefers SSH, he could replace it.

```suggestion
git clone https://github.com/mdn/content.git
```